deploy/crds/hco00.crd.yaml

```diff
@@ -75,6 +101,25 @@ spec:
                     description: Enable HyperV strict host checking for HyperV enlightenments
                       Defaults to true, even when HyperConvergedFeatureGates is empty
                     type: boolean
+                  liveMigration:
+                    default: true
+                    description: Enables VMIs to be live migrated. Without this, migrations
+                      are not possible and will be blocked
+                    type: boolean
+                  sidecar:
+                    default: true
+                    description: Enables using our sidecar hooks for injecting custom
+                      logic into the VMI startup flow. This is a very advanced feature
+                      that has security implications, which is why it is opt-in only
+                    type: boolean
+                  snapshot:
+                    default: true
+                    description: Enables the alpha offline snapshot functionality
```

Comment:
why do we want to expose this? Can a developer harm anything but their own data if they use this feature?